### DISCUSSION TITLE
ChatGPT web mode issue

### DISCUSSION THREAD

**Discussion by RaistlinSpringfield** [nextai-translator/nextai-translator#1394] [Q&A] [answered]
After updating to the latest version, it seems that the web mode pops new error message (screenshot attached). I have attempted the recommended method in the message but it did not work.

Any recommendations/suggestions would be greatly appreciated! Thanks!

![image](https://github.com/openai-translator/openai-translator/assets/165489765/ec61f63b-bd4a-4ea6-b7dd-80aee71cc2ce)

**Comment by yetone** [comment]
按照红字提示操作即可：打开 [ChatGPT](https://chat.openai.com) 的聊天页面，发送个消息。

Follow the instructions in red: Open the chat page of [ChatGPT](https://chat.openai.com) and send a message.

**Reply by RaistlinSpringfield** [reply, marked as answer]
I got it. Need to log out and re-log in ChatGPT website. It is resolved. Thanks!

**Reply by Ray-Lir** [reply]
你好，我尝试了上述行为，但此问题仍然存在。
Hi~ I tried the above behavior, but the problem still exists.
![image](https://github.com/openai-translator/openai-translator/assets/39153265/4070cef3-4f4a-4b76-bbd3-b1516ad1a1e4)
![image](https://github.com/openai-translator/openai-translator/assets/39153265/5023d548-f304-4139-bfb1-a1bf67837f78)

**Comment by zhl111** [comment]
我也解决不了这问题啊
